cx-portal-shared-components/src/components/content/Cards/AboutCard.tsx

```diff
@@ -0,0 +1,125 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 Mercedes-Benz Group AG and BMW Group AG
```

Comment:
Please remove company specific line.
I'm not sure how you're still using this header template. I updated the header template to only mention EF months ago:
https://github.com/eclipse-tractusx/portal-frontend/blob/dev/.vscode/settings.json#L14